src/App.js

```diff
@@ -9,14 +9,34 @@ import Home from "./pages/Home"
 import NotFound from "./pages/NotFound"
 import { Routes, Route } from "react-router-dom"
 import "./App.css"
-import mockCat from "./mockCat.js"
+// import mockCat from "./mockCat.js"
```

Comment:
You can remove this from the import. We'll only use it for the specs from here on out.